api/v1/models/MailingList.js

```diff
@@ -0,0 +1,27 @@
+var Model = require('./Model');
+var User = require('./User');
+var MailingListUser = require('./MailingListUser');
+
+var MailingList = Model.extend({
+	tableName: 'mailing_lists',
+	idAttribute: 'id'
+});
+
```

Comment:
Yeah so there's some refactoring to do to that list table for sure (wasn't really in the right frame of mind when I made it I suppose). 

There is no programmatic way to create a list, but that's primarily because I wanted to keep it as something managed in the db layer. Notice that the `utils/mail.py` has a mapping of all of the lists, and we'd lose that little bit of pseudo-type-safety if we could just add lists without updating that.